### PR TITLE
Fixed PREMIS rights issues

### PR DESCRIPTION
**Pull request by sarah-mason**
Updated `rights.csv` to remove incorrect use of citation column. Added a Statute right in `rights.csv` and updated the `rights-mets.xml` script to reflect the 4 examples in the csv. This now matches the rightsTransfer available in the sample-data under TestTransfers.

Very loosely related to[ issue 949](https://github.com/archivematica/Issues/issues/949) --where the first fix came from to update to PREMIS v. 3 while updating the METS schema references

### CODE REVIEW

File: user-manual/transfer/scripts/rights-mets.xml [hunk 1]
```diff
@@ -16,8 +16,8 @@
           <premis:licenseTerms>Terms of license.</premis:licenseTerms>
           <premis:licenseNote>migrate</premis:licenseNote>
           <premis:licenseApplicableDates>
-            <premis:startDate>2000-09-09</premis:startDate>
-            <premis:endDate>2010-09-08</premis:endDate>
+            <premis:startDate>09-09-2000</premis:startDate>
```
Comment (ross-spencer) [inline user-manual/transfer/scripts/rights-mets.xml:19]: I notice the date formatting switched around (LibreOffice?) I might have missed this in the sample-data, I guess there's not a big difference when it comes to trying to consume this data from the METS eventually? And clearly Archivematica doesn't have a problem with it.

**Review by ross-spencer — APPROVED** [review]
:ok_hand: Nice! 

(One comment about dates, but I think this is a useful follow-up to your work on the sample data. Thanks Sarah.)

**Comment by sallain** [comment]
@sromkey just tidying up some loose ends in this repo - do you think the comment above about date formatting matters? Otherwise this should be good to merge.

**Comment by sromkey** [comment]
@sallain I just found this languishing in my inbox, sorry! I don't take issue with the date thing, I would go ahead and merge.

**Review by replaceafill — APPROVED** [review]
Thank you @sarah-mason!

**Pull request merged**